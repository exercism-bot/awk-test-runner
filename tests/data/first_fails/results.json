{
  "version": 2,
  "status": "fail",
  "test-environment": {
    "gawk": "GNU Awk 5.1.0, API: 3.0 (GNU MPFR 4.1.0, GNU MP 6.2.1)",
    "bats": "Bats 1.7.0",
    "OS": "Ubuntu 22.04 LTS"
  },
  "tests": [
    {
      "name": "say one",
      "status": "fail",
      "test_code": "run bash first_fails.sh one\n[ \"$status\" -eq 0 ]\n[ \"$output\" == \"one\" ]",
      "message": "(in test file this_is_the_test_script.bats, line 6)\n  `[ \"$output\" == \"one\" ]' failed\n"
    },
    {
      "name": "say two",
      "status": "pass",
      "test_code": "run bash first_fails.sh two\n[ \"$status\" -eq 0 ]\n[ \"$output\" == \"two\" ]"
    },
    {
      "name": "say three",
      "status": "pass",
      "test_code": "run bash first_fails.sh three\n[ \"$status\" -eq 0 ]\n[ \"$output\" == \"three\" ]"
    }
  ]
}
