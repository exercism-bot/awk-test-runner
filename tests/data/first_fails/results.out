1..3
not ok 1 say one
# (in test file this_is_the_test_script.bats, line 6)
#   `[ "$output" == "one" ]' failed
ok 2 say two
ok 3 say three
